FirebaseTestProject/ViewModels/AuthViewModel.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  AuthViewModel.swift
+//  AuthViewModel
+//
+//  Created by Tim Miller on 8/12/21.
+//
+
+
+import FirebaseAuth
+import SwiftUI
+
+@MainActor class AuthViewModel: ObservableObject {
+  
+  private let auth = Auth.auth()
+  
+  @AppStorage("userSignedIn") var userSignedIn: Bool = false
+  
+  func getUser() -> String {
```

Comment:
Have this be a computed property with the name `userEmail` that returns a `String?`.